src/ducks/Studio/url.js

```diff
@@ -8,7 +8,20 @@ const { trendPositionHistory } = Events
 const COMPARE_CONNECTOR = '-CC-'
 const getMetricsKeys = metrics => metrics.map(({ key }) => key)
 const toArray = keys => (typeof keys === 'string' ? [keys] : keys)
-const convertKeyToMetric = (key, dict) => dict[key] || compatabilityMap[key]
+const convertKeyToMetric = (key, dict) =>
+  dict[key] || compatabilityMap[key] || getFromSubmetrics(key, dict)
+
+const getFromSubmetrics = (key, metrics) => {
+  for (let metricKey in metrics) {
+    let metric = metrics[metricKey]
+    if (metric && metric.subMetrics) {
+      const found = metric.subMetrics.find(
+        ({ key: subMetricKey }) => subMetricKey === key
+      )
+      if (found) return found
```

Comment:
`. find()` will return `undefined`, the same as the function's implicit `return`,  if the search was unsuccessful. The result can be returned in a single statement without unnecessary checks

Reply:
but we search over all metrics by key. need to check 'found' value